source/localizable/routing/redirection.md

```diff
@@ -22,12 +21,20 @@ currently in progress and start a new one, functioning essentially as a redirect
 If the new route has dynamic segments, you need to pass either a _model_ or an _identifier_ for each segment.
 Passing a model will skip the route's `model()` hook (since the model is already loaded).
 
+## The [`replaceWith()`][3] method
```

Comment:
I'm kind of disappointed that I couldn't draw up a simple situation where `replaceWith` is preferred to `transitionTo`; makes this part of the Guide resemble the API doc. Thing is, the example I know is too complicated and might end up extending this section for no good reason. If anyone has examples of when to use `replaceWith` please share with me and I'll update.
cc/ @locks @toddjordan @alexspeller